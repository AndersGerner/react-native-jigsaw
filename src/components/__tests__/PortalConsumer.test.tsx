import * as React from "react";
import renderer from "react-test-renderer";
import PortalConsumer from "../Portal/PortalConsumer";

it("renders portal consumer", () => {
  const tree = renderer.create(<PortalConsumer />).toJSON();

  expect(tree).toMatchSnapshot();
});
