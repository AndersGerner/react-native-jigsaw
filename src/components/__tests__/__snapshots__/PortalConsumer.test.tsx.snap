// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders portal consumer 1`] = `null`;
